src/system_database.ts

```diff
@@ -1683,10 +1681,41 @@ export class PostgresSystemDatabase implements SystemDatabase {
     return { workflows };
   }
 
-  async enqueueWorkflow(workflowId: string, queueName: string): Promise<void> {
+  async enqueueWorkflowInternal(
+    client: PoolClient,
+    workflowId: string,
+    queueName: string,
+    enqueueOptions?: EnqueueOptionsInternal,
+  ): Promise<void> {
+    const deDupId = enqueueOptions?.deDuplicationID ?? null;
+    console.log('In Enqueueing workflow', workflowId, queueName, deDupId);
+
+    try {
+      await client.query<workflow_queue>(
+        `
```

Comment:
nit: we don't need the full empty line at the start and end of this string literal